contracts/utils/test/lib_address_array.ts

```diff
@@ -23,22 +23,14 @@ describe('LibAddressArray', () => {
 
     before(async () => {
         await blockchainLifecycle.startAsync();
-    });
-    after(async () => {
-        await blockchainLifecycle.revertAsync();
-    });
-    before(async () => {
```

Comment:
I cloned this test to start my own, and noticed this inefficiency in there, and figured I'd leave it better off than I found it.